runtime/gateway.go

```diff
@@ -98,25 +99,19 @@ type Gateway struct {
 	localHTTPServer *HTTPServer
 	tchannelServer  *tchannel.Channel
 	tracerCloser    io.Closer
-	//	- panic ???
 	//	- process reporter ?
-
-	// Soon to be deprecated
-	Logger *zap.Logger
 }
 
 // DefaultDependencies are the common dependencies for all modules
 type DefaultDependencies struct {
-	// Logger is a server-scoped logger
-	// Deprecated: Use ContextLogger instead.
-	Logger *zap.Logger
 	// ContextExtractor extracts context for scope and logs field
 	ContextExtractor ContextExtractor
 	// ContextLogger is a logger with request-scoped log fields
 	ContextLogger ContextLogger
 	// ContextMetrics emit metrics from context
 	ContextMetrics ContextMetrics
 
+	Logger  *zap.Logger
```

Comment:
Intentionally removed the deprecated comment because `Logger` is passed to various constructors and should not be deprecated, example https://github.com/uber/zanzibar/pull/514/files#diff-2426caa827592efd9d55743dd4943e3cL70